.github/workflows/build_flang.yml

```diff
@@ -38,6 +38,9 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so the job can access it
       - uses: actions/checkout@v2
 
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
```

Comment:
Where is this defined?

Reply:
https://github.com/hendrikmuhs/ccache-action

Reply:
And here on the gh actions marketplace: https://github.com/marketplace/actions/ccache-for-gh-actions

Reply:
Is there a reason to use this custom cacching action over the Github-provided one (https://github.com/actions/cache)?

Reply:
Yes - the GitHub cache action only provides a way of caching, the use of you actually have to implement yourself. The ccache action used here provides that and builds on top of that - installs ccache itself, configures some options (the ones I mentioned in the opening comment), ensures ccache is available in the `$PATH`, creates a log entry with stats.

Reply:
Thanks!

Reply:
Only concern is using an action in a private individual's repo.

WDYT, @bryanpkc @shivaramaarao

Reply:
The TypeScript code looks pretty straightforward. Does the presence of the action in the GitHub Marketplace ensure its continued availability? Will it disappear suddenly and break our CI?

Reply:
I doubt presence on the actions marketplace guarantees anything. If that's a concern for you, you may consider forking the project into `flang-compiler` organisation. Personally though I wouldn't recommend that, this PR is quite small and self-contained. Should the action ever be removed, fallback to previous version is easy enough.

Alternatively, ccache installation and caching can be rewritten in bash so that you don't need to rely on external actions, but I'm not keen on doing that when current solution seems sufficient.

Reply:
@janisozaur I agree with your view. I don't have a concern with this use.